joomla/joomla-cms
# [5.0] joomla-tab corners

### Summary of Changes
Makes sure that all 4 corners are rounded


### Testing Instructions
either apply pr and `run npm build:css` or use a pre-built install

Go to anything using the joomla-tabs element eg new article and check the 4 corners to make sure they are all rounded


### Actual result BEFORE applying this Pull Request
Screenshot in dark mode just for clarity
![image](https://github.com/joomla/joomla-cms/assets/1296369/a5203927-fe1a-4768-8b43-5a0b24e8edaf)

![image](https://github.com/joomla/joomla-cms/assets/1296369/c02eca3d-f76b-4fb5-9385-8745683a777b)



### Expected result AFTER applying this Pull Request
Screenshot in dark mode just for clarity

![image](https://github.com/joomla/joomla-cms/assets/1296369/2a607860-2a12-4529-8b38-1c07625fdf1e)

![image](https://github.com/joomla/joomla-cms/assets/1296369/3ca007c8-9743-49c9-b6f4-e7d04b7af3e4)



### Link to documentations
Please select:
